snapshots/output/basic-project/lib/other.dart

```diff
@@ -26,34 +26,30 @@
 //  ^^^ definition scip-dart pub dart_test 1.0.0 lib/other.dart/Foo#<constructor>().
 //  documentation ```dart
 //  ^^^ reference scip-dart pub dart_test 1.0.0 lib/other.dart/Foo#
-//      ^^^^ reference local 0
-//           ^^^^ definition local 1
-//           documentation ```dart
+//           ^^^^ reference local 0
```

Comment:
We were previously always calculating a definition for parameters, even formal ones referring to instance variables, now we aren't, hence the removal of `definition local 1`

`_far` is a private instance variable, so the reference is just a `local 0`